types/ember-qunit/ember-qunit-tests.ts

```diff
@@ -7,159 +6,132 @@ import {
     skip,
     only,
     todo,
-    moduleFor,
-    moduleForModel,
-    moduleForComponent,
     setResolver,
     setupRenderingTest,
     setupTest,
 } from 'ember-qunit';
-import { TestContext } from 'ember-test-helpers';
+import { render, TestContext } from '@ember/test-helpers';
```

Comment:
The `TestContext` move is going to be a fun one to deal with in consuming codebases 🥲

Reply:
Alas, yes. I thought about re-exporting, but at some point people are going to have to bite the bullet, and no one should actually be using `ember-test-helpers` at runtime anymore, so… 🤷🏼‍♂️